gpu/Cargo.toml

```diff
@@ -2,16 +2,9 @@
 name = "ya-self-test-gpu"
 version = "0.1.0"
 edition = "2021"
+publish = false
 
-[lib]
-path = "src/lib.rs"
-
-[[bin]]
-name = "ya-self-test-gpu"
-path = "src/main.rs"
 
 [dependencies]
-anyhow = "1.0"
-parse-size = "1.0.0"
-roxmltree = "0.18.0"
+golem-gpu-info = { git = "https://github.com/golemfactory/golem-gpu-info.git", branch = "feature/multi-gpu"}
```

Comment:
This library is a copy of https://github.com/golemfactory/ya-runtime-ai/tree/main/gpu-detection 
It supports GPU `uuid` parameter, and if given none it tries to access the first available Nvidia GPU. It was left like this because multi GPU support was planned to be done in a near future.
Original `ya-self-test-gpu` probably has not been tested on host with multiple GPUs, but it just tries list all info about GPUs, and it parses output looking for some specific lines.
These libraries can behave differently on multi GPUs hosts.

Reply:
yep, look at branch `feature/multi-gpu`. i changed your code to iterate over gpu. it was tested in cloud on 2xNVIDIA A30 machine. and i made regresion on 'NVIDIA GeForce RTX 3090' on our testnet node `0x9747052ae07cdfa96f16fbbb92491284cd160b29` with 1xNVIDIA GeForce RTX 3060.

Reply:
If we do not want to have here a revision/tag of the `golem-gpu-info` dependency then it is ok.